crates/redstone/src/core/config.rs

```diff
@@ -1,6 +1,11 @@
+use crate::{
+    helpers::slice::has_duplicates, network::error::Error, FeedId, SignerAddress, TimestampMillis,
+};
 use alloc::vec::Vec;
 
-use crate::{FeedId, SignerAddress, TimestampMillis};
+/// MAX_SIGNERS_COUNT describes maximum number of signers in Config.
+const MAX_SIGNERS_COUNT: usize = u8::MAX as usize;
```

Comment:
`signer_count`, without `s`, like Psikuta

https://english.stackexchange.com/questions/11350/use-of-the-plural-with-list-and-when-item-count-is-0